samples/Http2SampleApp/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM microsoft/dotnet:2.0.0-runtime-stretch
+
+RUN apt-get update && \
+    apt-get install -y --no-install-recommends libssl-dev && \
```

Comment:
Why libssl-dev. Is libssl1.0.0 insufficient?

Reply:
It may not be, I'll try that :).